icon-descriptions.yml

```diff
@@ -4,6 +4,10 @@ AirCon: Air condition
 Airplane: Reise
 AirplaneHotel: Fly og hotell
 Alert: Utropstegn
+AlertError16: Rødt utropstegn
```

Comment:
I'm not sure we should add 16 here as I think we append the size number in the import script somewhere, no?

Reply:
Yep you're right! My bad! 🙈

Reply:
Fixed on main